api/lib/ModelHelpers.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
+var moment = require('moment-timezone');
+moment.tz.setDefault("Europe/London");
```

Comment:
see previous about setting a global timezone somewhere - if this needs to be done with some environment variable passed to node?